index.js

```diff
@@ -1,10 +1,11 @@
 var deselectCurrent = require('toggle-selection');
 
 function copy(text, options) {
-  var debug, message, cb, reselectPrevious, range, selection, mark;
+  var debug, copyKey, message, cb, reselectPrevious, range, selection, mark;
   if (!options) { options = {}; }
   debug = options.debug || false;
-  message = options.message || 'Copy to clipboard: Ctrl+C, Enter';
+  copyKey = /mac os x/i.test(navigator.userAgent) ? '⌘' : 'Ctrl';
```

Comment:
Good call. Done in https://github.com/sudodoki/copy-to-clipboard/commit/a3e3d1547d5f5d65659cc621cdcdcaf5c3720510.